components/Tabs/src/private/MDCItemBar.m

```diff
@@ -489,6 +490,9 @@ - (void)stopObservingItems {
 }
 
 - (UIUserInterfaceSizeClass)horizontalSizeClass {
+  if (self.tabBar && self.tabBar.sizeClassDelegate && [self.tabBar.sizeClassDelegate respondsToSelector:@selector(horizontalSizeClassForObject:)]) {
```

Comment:
The check to `self.tabBar` is not needed since if it's nil the sizeClassDelegate check will be nil and fail automatically. So it's implicitly checked by the later arguments.

This could be simplified to:

```
if ([self.tabBar.sizeClassDelegate respondsToSelector:@selector(horizontalSizeClassForObject:)] {

}
```

And it would have the same effect.

Reply:
Done, thanks!